SentryTestUtils/TestDisplayLinkWrapper.swift

```diff
@@ -53,46 +57,53 @@ public class TestDisplayLinkWrapper: SentryDisplayLinkWrapper {
     }
     
     public func normalFrame() {
-        internalTimestamp += currentFrameRate.tickDuration
+        dateProvider.advance(by: currentFrameRate.tickDuration)
         call()
     }
     
-    public func fastestSlowFrame() {
-        internalTimestamp += slowFrameThreshold(currentFrameRate.rawValue) + timeEpsilon
+    public func fastestSlowFrame() -> CFTimeInterval {
+        let duration: Double = slowFrameThreshold(currentFrameRate.rawValue) + timeEpsilon
+        dateProvider.advance(by: duration)
         call()
+        return duration
```

Comment:
Return the duration so that we can know what to assert on in the profiling tests; we record the duration of frames, so need to make sure the serialized value is as expected.

This required adding a few `_ = ` to other usages of this api in some tests.